src/Benchmarks/Data/ApplicationDbContext.cs

```diff
@@ -9,6 +9,6 @@ public class ApplicationDbContext : DbContext
     {
         public DbSet<World> World { get; set; }
 
-        public DbSet<Fortune> Fortunue { get; set; }
+        public DbSet<Fortune> Fortune { get; set; }
```

Comment:
I ended up fixing this as it touched a lot more places (I misnamed the migration too :frowning:)